INSTALL.md

```diff
@@ -1,46 +1,116 @@
-$ pip install git+git://github.com/rosskarchner/govdelivery
+After installation, Homebrew will output instructions similar to:
+```bash
+To finish the installation, source activate.sh in your shell:
+  source /Users/[YOUR MAC OSX USERNAME]/homebrew/opt/autoenv/activate.sh
 ```
 
-## Back-end environment variables
+Any time you run the project you'll need to run that last line. If you'll be working with
+the project consistently, we suggest adding it to your bash profile by running:
+```bash
+echo 'source /Users/[YOUR MAC OSX USERNAME]/homebrew/opt/autoenv/activate.sh' >> ~/.bash_profile
+```
 
-The project needs a number of environment variables.
-The project uses a WordPress API URL to pull in content
-and GovDelivery for running the subscription forms:
+If you need to find this info again later, you can run:
+```bash
+brew info autoenv
+```
 
-- `WORDPRESS` (URL to WordPress install)
-- `GOVDELIVERY_BASE_URL`
-- `GOVDELIVERY_ACCOUNT_CODE` (GovDelivery account variable)
-- `GOVDELIVERY_USER` (GovDelivery account variable)
-- `GOVDELIVERY_PASSWORD` (GovDelivery account variable)
-- `SUBSCRIPTION_SUCCESS_URL` (Forwarding location on Subscription Success)
+**NOTE:** If you use ZSH you'll need to use
+[zsh-autoenv](https://github.com/Tarrasch/zsh-autoenv), but we can't provide support for issues that may arise.
 
-> You can also export the above environment variables to your `.bash_profile`,
-or use your favorite alternative method of setting environment variables.
+### Elasticsearch
 
-**NOTE:** GovDelivery is a third-party web service that powers our subscription forms.
-Users may decide to swap this tool out for another third-party service.
-The application will function but throw an error if the above GovDelivery values are not set.
+[Install Elasticsearch](http://www.elastic.co/guide/en/elasticsearch/reference/current/setup.html)
+however you'd like. (We use [Homebrew](http://brew.sh)):
+```bash
+brew install elasticsearch
+```
+
+Just as with autoenv, Homebrew will output similar instructions after installation:
+```bash
+To have launchd start elasticsearch at login:
+    ln -sfv /Users/[YOUR MAC OSX USERNAME]/homebrew/opt/elasticsearch/*.plist ~/Library/LaunchAgents
+Then to load elasticsearch now:
+    launchctl load ~/Library/LaunchAgents/homebrew.mxcl.elasticsearch.plist
+Or, if you don't want/need launchctl, you can just run:
+    elasticsearch --config=/Users/[YOUR MAC OSX USERNAME]/homebrew/opt/elasticsearch/config/elasticsearch.yml
+```
+
+Any time you work on the project, you'll need to open a new tab and run that last line.
+If you'll be working on the project consistently, we suggest using the first option
+utilizing `launchd`.
+
+If you need to find this info again later, you can run:
+```bash
+brew info elasticsearch
+```
+
+### Sheer
+
+To [install sheer](https://github.com/cfpb/sheer#installation), start by cloning the
+Sheer GitHub project to wherever you keep your projects (not inside cfgov-refresh):
```

Comment:
Possibly change `not inside cfgov-refresh` > `not inside the cfgov-refresh directory`